<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSelector</class>
 <widget class="QWidget" name="ImageSelector">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>497</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QImageLister" name="lstImages"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblImageCount">
     <property name="text">
      <string>Total Images: 0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cluster Images</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QImageLister</class>
   <extends>QListWidget</extends>
   <header>QImageLister.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
